src/Microsoft.AspNet.Mvc.TagHelpers/LinkTagHelper.cs

```diff
@@ -365,5 +359,22 @@ private void BuildLinkTag(TagHelperAttributeList attributes, TagHelperContent bu
 
             builder.Append("/>");
         }
+
+        private enum Mode
+        {
+            /// <summary>
+            /// Just adding a file version for the generated urls.
+            /// </summary>
+            FileVersion = 0,
+            /// <summary>
```

Comment:
While here please add blank lines to separate the `enum` values here and in `ScriptTagHelper`